src/ext/java/org/opentripplanner/ext/vectortiles/layers/stops/DigitransitStopPropertyMapper.java

```diff
@@ -34,20 +34,31 @@ protected static DigitransitStopPropertyMapper create(
 
   @Override
   protected Collection<KeyValue> map(RegularStop stop) {
-    Collection<TripPattern> patternsForStop = transitService.getPatternsForStop(stop);
+    return getBaseKeyValues(stop, i18NStringMapper, transitService);
+  }
 
-    String type = patternsForStop
-      .stream()
-      .map(TripPattern::getMode)
-      .collect(Collectors.groupingBy(Function.identity(), Collectors.counting()))
-      .entrySet()
-      .stream()
-      .max(Map.Entry.comparingByValue())
-      .map(Map.Entry::getKey)
-      .map(Enum::name)
-      .orElse(null);
+  protected static Collection<KeyValue> getBaseKeyValues(
+    RegularStop stop,
+    I18NStringMapper i18NStringMapper,
+    TransitService transitService
+  ) {
+    return List.of(
+      new KeyValue("gtfsId", stop.getId().toString()),
+      new KeyValue("name", i18NStringMapper.mapNonnullToApi(stop.getName())),
+      new KeyValue("code", stop.getCode()),
+      new KeyValue("platform", stop.getPlatformCode()),
+      new KeyValue("desc", i18NStringMapper.mapToApi(stop.getDescription())),
+      new KeyValue(
+        "parentStation",
+        stop.getParentStation() != null ? stop.getParentStation().getId() : null
```

Comment:
This is a breaking change. This shouldn't be shared between the mappers as the old one always returned a string.